vc.adoc

```diff
@@ -1100,6 +1100,7 @@ values, therefore this way is deprecated in 2.0 and is removed in 3.0.
 |items[].episode.season |Integer |true |Number of season.
 |items[].thumbnail |String |false |Thumbnail of the Video.
 |items[].customInfo |Map |false |Array of Key-Value to represent additional information of an entity. Partners could contain diverse information of same entities. This type designed to store that information.
+|items[].attachments |Map |true |Array of Key-Value to represent the title & url of an "attachment".
```

Comment:
@ParitoshikPaul it's not a Map, it's Array[Object], see how `items` is described.

You have to add each attribute (attachments, title, URL) on a separate line